react/Checkbox/Checkbox.js

```diff
@@ -22,10 +22,10 @@ export default class Checkbox extends Component {
 
   static propTypes = {
     id: PropTypes.string.isRequired,
-    label: PropTypes.string.isRequired,
```

Comment:
Well there you go, a string is also valid for an element prop type.  One last thing - I realise it wouldn't make any difference to how it's evaluated but would it still be worth making this a oneOf string or element prop type for the sake of documenting the fact that a string is valid input?